_layouts/blog.html

```diff
@@ -1,16 +1,22 @@
 ---
-layout: default
+layout: page
```

Comment:
There is an SVG symbolic version of MyPaint's Logo under `/assets/site/2016-0320-Site_Logo_Symbolic.svg`. You could load that and use [css to manipulate its fill color](https://css-tricks.com/using-svg/).